anvil/src/udev.rs

```diff
@@ -95,6 +95,15 @@ use smithay::{
 };
 use tracing::{debug, error, info, trace, warn};
 
+// we cannot simply pick the first supported format of the intersection of *all* formats, because:
+// - we do not want something like Abgr4444, which looses color information, if something better is available.
+// - some formats might perform terribly.
+// - we might need some work-arounds, if one supports modifiers, but the other does not.
+// - we can't handle formats with small alpha channels yet in the renderer like `Abgr2101010`.
+//
+// So lets just pick `ARGB8888`/`ABGR8888` for now, they are widely supported.
+const SUPPORTED_FORMATS: &[Fourcc] = &[Fourcc::Abgr8888, Fourcc::Argb8888];
```

Comment:
Does this intentionally not include 10bit variants? Winit has been changed to prefer 10bit.

Reply:
Oh, then I need to drop the winit changes. 10-bit isn't ready yet in this PR.